packages/react-components/src/components/VideoTile.tsx

```diff
@@ -229,6 +261,27 @@ export const VideoTile = (props: VideoTileProps): JSX.Element => {
   const canShowLabel = showLabel && (displayName || (showMuteIndicator && isMuted));
   const participantStateString = participantStateStringTrampoline(props, locale);
 
+  /* @conditional-compile-remove(pinned-participants) */
+  const optionsMenuItems = (menuItems: VideoTileMenuItems): IContextualMenuItem[] => {
+    const contextualMenuItems: IContextualMenuItem[] = [];
+    menuItems.forEach((item: VideoTileMenuItem) => {
```

Comment:
nit: this is a great case for the `map` function. Benefits are shorter code but also let the reader know that the `contextualMenuItems` correlate 1:1 with the props.menuItems:

```
const contextualMenuItems: IContextualMenuItem[] = menuItems.map(menuItem => ({
  key: menuItem.key,
  text: item.text,
  onClick: item.onClick,
  iconProps: item.icon
});
```